content/posts/how-to-evolve-application-add-quality/index.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "How to evolve an application to make it more robust and maintainable - Add quality "
+description = ""
+date = 2023-05-29
+featured = false
+draft = false
+comment = true
+toc = true
+reward = true
+pinned = false
+categories = [
+"quality", "test", "CICD", "Sonarqube", "Owasp", "Gradle"
+]
+tags = [
+""
+]
+series = [
+" How to evolve an application to make it more robust and maintainable"
+]
+images = []
++++
+
+<!--more-->
+
+
+## Foreword
+
+In our last articles we looked at how to add tests and how to run them automatically via the CI/CD (GitHub Actions).
+In this article we'll continue to improve the quality of our project using Sonarqube.
+
+
+
+## What is Sonarcloud or Sonarqube
+[index.md](..%2FHowTo%2Fhow-to-use-sonarcloud%2Findex.md)
```

Comment:
Please can you correctly link this file?

Reply:
Done